Formula/git-hound.rb

```diff
@@ -0,0 +1,56 @@
+class GitHound < Formula
+  desc "Git plugin that prevents sensitive data from being committed"
+  homepage "https://github.com/ezekg/git-hound"
+  url "https://github.com/ezekg/git-hound/archive/1.0.0.tar.gz"
+  sha256 "32f79f470c790db068a23fd68e9763b3bedc84309a281b4c99b941d4f33f5763"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
Does the project not use go modules?

Reply:
I just updated it to use Go modules yesterday, before making this PR. Is this line not required? Sorry — first time making a PR here so I kind of based my install on a couple other Go formulas like Caddy.

Reply:
You probably only need something like telegraf: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/telegraf.rb

Reply:
Updated 👍